core/src/components/textarea/textarea.scss

```diff
@@ -149,6 +151,7 @@
   outline: none;
 
   background: transparent;
+  color: inherit;
```

Comment:
What does this do?

Reply:
Good question - I think I was adding it during testing and didn't clean it up after. I'll test more after my early meetings and confirm. 

It looks like the cursor color also needs to be updated here as well.